app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -3,6 +3,16 @@ module AuthnJwt
 
     ValidateStatus = CommandClass.new(
       dependencies: {
+        fetch_signing_key: ::Util::ConcurrencyLimitedCache.new(
```

Comment:
I think that it worth to align the parameter name in different classes https://github.com/cyberark/conjur/blob/c4eedc7b95ee1fb5bd5a9581300614f13bbe880c/app/domain/authentication/authn_jwt/validate_and_decode/validate_and_decode_token.rb#L10
CC: @tzheleznyak

Reply:
@nessiLahav  rebase the bugfix branch to you branch